src/docs/ftw-styling/how-to-improve-performance/index.md

```diff
@@ -100,26 +95,3 @@ sparse attributes. This is a feature FTW has not yet leveraged fully,
 but it is created to reduce unnecessary data and speed up rendering. You
 can read more from
 [Marketplace API reference for sparse attributes](https://www.sharetribe.com/api-reference/#sparse-attributes).
-
-## About code-splitting
```

Comment:
I'd suggest that we say something short about code splitting here. 

E.g. "Code splitting is enabled with Loadable Components and by default route-based splits are made through src/routeConfiguration.js. If you want to improve performance, you should prefer subcomponents inside page-directories instead of adding more code to shared components directory. Those components end up to main chunk file that is downloaded on each page (when full page-load is requested)."